sse2neon.h

```diff
@@ -106,6 +106,14 @@
 #pragma message("Macro name collisions may happen with unsupported compilers.")
 #endif
 
+#if defined(__GNUC__) && !defined(__clang__)
+#define FORCE_INLINE_NO_OPTIMIZE static inline __attribute__((optimize("O0")))
```

Comment:
Let's decouple `__attribute__((optimize("O0")))` or `__attribute__((optnone))` out of `FORCE_INLINE`. That is, there should be a new macro `SSE2NEON_OPTNONE` to suppress the optimizations for certain functions.

Reply:
`always_inline` conflicts with `SSE2NEON_OPTNONE`, so the suppressed funcs will be like
```c
static inline SSE2NEON_OPTNONE __m128i _mm_cvtpd_epi32(__m128d a)
```

Not sure is it better to merge all the attributes together?

Reply:
> `always_inline` conflicts with `SSE2NEON_OPTNONE`, so the suppressed funcs will be like

Got it. We then need new macro `FORCE_INLINE_OPTNONE` , which should be specified via `##pragma push_macro`. `OPTNONE` is shorter than `NO_OPTIMIZE`.